notebooks/tests/__init__.py

```diff
@@ -50,7 +51,7 @@ def _remove_folders(input_folders: [str]) -> None:
 
 
 @contextmanager
-def start_nessie() -> subprocess.Popen:
+def start_nessie() -> Iterator[subprocess.Popen]:
```

Comment:
this is still not the most precise type for a contextmanager method but at least more accurate than before